src/style/color-palette.scss

```diff
@@ -1,271 +1,271 @@
-    --contrast-1700: #fff;
-
-    --color-red-lighter: #ef9a9a;
-    --color-red-light: #ef5350;
-    --color-red-default: #e53935;
-    --color-red-dark: #c62828;
+    --contrast-1000: #757580;
+    --contrast-1100: #5e5e6c;
+    --contrast-1200: #474756;
+    --contrast-1300: #303042;
+    --contrast-1400: #272739;
+    --contrast-1500: #232335;
+    --contrast-1600: #19192c;
+    --contrast-1700: #000;
+
+    // Extended color palette that automatically get slightly darker in darkmood
+    --color-red-lighter: #ffcdd2;
+    --color-red-light: #e57373;
+    --color-red-default: #f44336;
+    --color-red-dark: #d32f2f;
     --color-red-darker: #b71c1c;
 
-    --color-pink-lightest: #f48fb1;
-    --color-pink-light: #ec407a;
-    --color-pink-default: #d81b60;
-    --color-pink-dark: #ad1457;
+    --color-pink-lightest: #f8bbd0;
+    --color-pink-light: #f06292;
+    --color-pink-default: #e91e63;
+    --color-pink-dark: #c2185b;
     --color-pink-darker: #880e4f;
 
-    --color-purple-lighter: #ce93d8;
-    --color-purple-light: #ab47bc;
-    --color-purple-default: #8e24aa;
-    --color-purple-dark: #6a1b9a;
+    --color-purple-lighter: #e1bee7;
+    --color-purple-light: #ba68c8;
+    --color-purple-default: #9c27b0;
+    --color-purple-dark: #7b1fa2;
     --color-purple-darker: #4a148c;
 
-    --color-deep-purple-lighter: #b39ddb;
-    --color-deep-purple-light: #7e57c2;
-    --color-deep-purple-default: #5e35b1;
-    --color-deep-purple-dark: #4527a0;
+    --color-deep-purple-lighter: #d1c4e9;
+    --color-deep-purple-light: #9575cd;
+    --color-deep-purple-default: #673ab7;
+    --color-deep-purple-dark: #512da8;
     --color-deep-purple-darker: #311b92;
 
-    --color-indigo-lighter: #9fa8da;
-    --color-indigo-light: #5c6bc0;
-    --color-indigo-default: #3949ab;
-    --color-indigo-dark: #283593;
+    --color-indigo-lighter: #c5cae9;
+    --color-indigo-light: #7986cb;
+    --color-indigo-default: #3f51b5;
+    --color-indigo-dark: #303f9f;
     --color-indigo-darker: #1a237e;
 
-    --color-blue-lighter: #90caf9;
-    --color-blue-light: #42a5f5;
-    --color-blue-default: #1e88e5;
-    --color-blue-dark: #1565c0;
+    --color-blue-lighter: #bbdefb;
+    --color-blue-light: #64b5f6;
+    --color-blue-default: #2196f3;
+    --color-blue-dark: #1976d2;
     --color-blue-darker: #0d47a1;
 
-    --color-light-blue-lighter: #81d4fa;
-    --color-light-blue-light: #29b6f6;
-    --color-light-blue-default: #039be5;
-    --color-light-blue-dark: #0277bd;
+    --color-light-blue-lighter: #b3e5fc;
+    --color-light-blue-light: #4fc3f7;
+    --color-light-blue-default: #03a9f4;
+    --color-light-blue-dark: #0288d1;
     --color-light-blue-darker: #01579b;
 
-    --color-cyan-lighter: #80deea;
-    --color-cyan-light: #26c6da;
-    --color-cyan-default: #00acc1;
-    --color-cyan-dark: #00838f;
+    --color-cyan-lighter: #b2ebf2;
+    --color-cyan-light: #4dd0e1;
+    --color-cyan-default: #00bcd4;
+    --color-cyan-dark: #0097a7;
```

Comment:
This looks alarming, but it's actually just git matching the light mode of the updated file to the dark mode of the old file, because the indentation changed… 😅